### DISCUSSION TITLE
I have a new feature suggestion for Glow.

### DISCUSSION THREAD

**Discussion by OpenSoftware-World** [turkaysoft/glow#11] [Ideas]
Hello, as LinuxUsersLinuxMint, I have a nice feature suggestion for your glow software. Can you add language support to the Glow software, for example, the user can create a file called "chinese.ini" and write Chinese texts and save it to the program. @roines45

**Comment by turkaysoft** [comment]
Hello,

I have a draft on a dynamic language detection system.
That is, the user will be able to add the language file they want and change the language without interfering with the source code.

In this way, the interface language can be changed without interfering with the code.

Tests are going on right now.

I can develop and release it with version 24.07 or 24.08.
